src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Categories/Blocks/form.html.twig

```diff
@@ -123,6 +123,28 @@
 
       {{ renderhook('displayBackOfficeCategory') }}
 
+      <div class="form-group row">
+        <label class="form-control-label"></label>
+        <div class="col-sm">
+          <h2>{{ 'Search Engine Optimization'|trans({}, 'Admin.Catalog.Feature') }}</h2>
+          <p class="mb-2">
+            {{ ' Improve your ranking and how your category page will appear in search engines results.'|trans({}, 'Admin.Catalog.Feature') }}
+          </p>
+          <p>{{ "Here is a preview of your search engine result, play with it!"|trans({}, 'Admin.Catalog.Feature') }}</p>
```

Comment:
Is it the right form of quotation marks here?

Reply:
Good catch, it's not. :)